### ISSUE TITLE
`sideWatch` is not a function in  `@embroider/broccoli-side-watch@1.0.0`

### ISSUE THREAD

**Issue by aklkv** [embroider-build/embroider#2147]
After upgrading to `v1.0.0` I started seeing:

```
sideWatch is not a function
```

setup is has not changes:

```js
const sideWatch = require('@embroider/broccoli-side-watch');

trees: {
      app: sideWatch('app', { watching: ['../addons'] }),
    },
```

**Comment by ijlee2** [comment]
I also ran into the issue. The error goes away if we write,

```js
/* ember-cli-build.js */
const { default: sideWatch } = require('@embroider/broccoli-side-watch');
```

but suspect that this change was unintentional.

**Comment by simonihmig** [comment]
Fixed in 1.0.1! https://github.com/embroider-build/embroider/releases/tag/v1.0.1-%40embroider%2Fbroccoli-side-watch

Apologies for the trouble!

**Comment by aklkv** [comment]
No worries! Thank you for the quick fix 🙏🏻 I though I would report right away since it was released as stable V1
